plugins/@grouparoo/salesforce/src/lib/export/mapping.ts

```diff
@@ -86,22 +86,22 @@ const supportedTypeMap = {
   id: "string",
   picklist: "string",
   currency: "float",
+  masterrecord: "string", // When records are merged, the ID of the record that is saved
+  reference: "string", // Cross-references to a different object. Analogous to a foreign key field in SQL.
+  base64: "string", // Base 64-encoded binary data.
   //TODO:
-  address: null,
+  address: null, //A compound data type that contains address field data.
   date: null,
```

Comment:
Dates are probably the most important ones to fix on all these plugins, especially in Salesforce. For example, `CloseDate` is a required field when making an `Oppurtunity`